lib/core/src/model.rs

```diff
@@ -1119,6 +1121,8 @@ pub struct RefundableSwap {
     pub timestamp: u32,
     /// Amount that is refundable, from all UTXOs
     pub amount_sat: u64,
+    /// The txid of an existing pending refund tx, if any
+    pub pending_refund_tx_id: Option<String>,
```

Comment:
This might also be the refund_tx_id of the last refund right? If there are multiple txs to the lockup address, so it might not be pending...

Reply:
I had thought about that case, but had dismissed it as an edge case for which we are fine with potentially providing incorrect info (i.e. exposing as "pending" when it's confirmed). To fully support that case, we would need to track and expose a collection of refund txs. Maybe it isn't worth it for now, and if so, I think labelling this tx id as pending is fine, as it's the most common case. Wdyt?

Reply:
I'm fine with it

Reply:
Or `last_refund_tx_id`

Reply:
Nice! [026ca36](https://github.com/breez/breez-sdk-liquid/pull/688/commits/026ca3625dcda2ae5cf064b71f74ff0b558bfecd)